.github/workflows/keep-alive.yaml

```diff
@@ -0,0 +1,18 @@
+name: Keep alive
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 1 * *' # monthly
+
+jobs:
+  snap:
+    uses: canonical/robotics-actions-workflows/.github/workflows/keepalive.yaml@main
+    with:
+      workflow_files: "./snap.yaml, ./snap-lxc.yaml, ./multi-snap.yaml, ./multi-snap-lxc.yaml, ./multi-snap-lxc-self-hosted.yaml"
```

Comment:
I get that this is an example but none of those workflows are scheduled...
How about pointing to the `check-cves.yaml` file from #4 instead ?

Reply:
Pointed to check-cves.yaml instead